addons/addon-base-raas/packages/base-raas-services/lib/environment/built-in/environment-url-service.js

```diff
@@ -58,7 +57,16 @@ class EnvironmentUrlService extends Service {
     const credentials = `${username}\n${password}`;
     const publicKey = await request(rstudioPublicKeyUrl);
     const [exponent, modulus] = publicKey.split(':', 2);
-    const params = { v: rstudioEncryptor.encrypt(credentials, exponent, modulus) };
+    const exponentBuffer = Buffer.from(exponent, 'hex');
+    const modulusBuffer = Buffer.from(modulus, 'hex');
+    const key = new NodeRSA();
+    const publicKeyObject = key.importKey({ n: modulusBuffer, e: exponentBuffer }, 'components-public');
+    const payloadBuffer = Buffer.from(credentials);
+    const result = crypto.publicEncrypt(
+      { key: publicKeyObject.exportKey('public'), padding: crypto.constants.RSA_PKCS1_PADDING },
+      payloadBuffer,
+    );
+    const params = { v: result.toString('base64') };
```

Comment:
was rStudioEncryptor.encrypt returning base64 string before?

Reply:
If we don't use base64 string encoding here, we get an empty query string encoded result. 

For example:
`https://rstudio-<workspaceId>.<custom-domain>/auth-do-sign-in?v=`